Classes/Controller/Backend/ControlCenter/TemplaVoilaPlus8UpdateController.php

```diff
@@ -1007,7 +1016,8 @@ protected function convertAllDsTo(array $allDs, array $allTos, string $packageNa
 
             GeneralUtility::writeFile(
                 $publicExtensionDirectory . $innerPathes['mappingConfiguration'][$scopeName] . '/' . $yamlFileName,
-                \Symfony\Component\Yaml\Yaml::dump($mappingConfiguration, 100, 4, \Symfony\Component\Yaml\Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK) // No inline style please
+                \Symfony\Component\Yaml\Yaml::dump($mappingConfiguration, 100, 4, \Symfony\Component\Yaml\Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK)
+                // No inline style please
```

Comment:
This comment is a hint to the Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK ... now it misses its context